src/attestation/azext_attestation/generated/_help.py

```diff
@@ -0,0 +1,98 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['attestation'] = """
+    type: group
+    short-summary: attestation
+"""
+
+helps['attestation list'] = """
+    type: command
+    short-summary: Returns a list of attestation in a subscription or resource group.
+    examples:
+      - name: List all attestations in a subscription
+        text: |-
+               az attestation list
+      - name: List all attestations in a resource group
+        text: |-
+               az attestation list --resource-group "MyResourceGroup"
+"""
+
+helps['attestation show'] = """
+    type: command
+    short-summary: Gets the status of an attestation.
+    examples:
+      - name: Get details for an attestation in a subscription
+        text: |-
+               az attestation show --name "myattestationprovider" --resource-group "MyResourceGroup"
+"""
+
+helps['attestation create'] = """
+    type: command
+    short-summary: Creates or updates an attestation.
+    parameters:
+      - name: --policy-signing-certificates-keys
+        short-summary: The value of the "keys" parameter is an array of JWK values.  Bydefault, the order of the JWK va\
```

Comment:
Does this show up on the command line with attestation create -h?
If so, this very verbose for command line display. Maybe this should move to the help file.